src/main/cluster.ts

```diff
@@ -170,26 +170,30 @@ export class Cluster implements ClusterModel, ClusterState {
       await this.refreshAllowedResources()
       this.isAdmin = await this.isClusterAdmin()
       this.ready = true
-      this.kubeCtl = new Kubectl(this.version)
-      this.kubeCtl.ensureKubectl() // download kubectl in background, so it's not blocking dashboard
+      this.ensureKubectl()
```

Comment:
This is dangerous, because if the kubectl promise throws then we will get a "UnhandledPromiseRejectionError" which in the future will cause Lens to crash.